### PR TITLE
[FIX/IMP] a couple issues observed during startup, take 2 :)

### PR DESCRIPTION
**Pull request by sbidoul**
Hopefully this one is correct.

@ThomasBinsfeld @guewen

### CODE REVIEW

File: queue_job/jobrunner/runner.py [hunk 1]
```diff
@@ -326,9 +327,9 @@ def select_jobs(self, where, args):
                  "priority, EXTRACT(EPOCH FROM eta), state "
                  "FROM queue_job WHERE %s" %
                  (where, ))
-        with closing(self.conn.cursor()) as cr:
+        with closing(self.conn.cursor("select_jobs", withhold=True)) as cr:
```
Comment (guewen) [inline queue_job/jobrunner/runner.py:330]: Nice, I didn't know about the `withhold` parameter

File: queue_job/jobrunner/runner.py [hunk 2]
```diff
@@ -386,8 +387,9 @@ def initialize_databases(self):
                 _logger.debug('queue_job is not installed for db %s', db_name)
             else:
                 self.db_by_name[db_name] = db
-                for job_data in db.select_jobs('state in %s', (NOT_DONE,)):
-                    self.channel_manager.notify(db_name, *job_data)
+                with db.select_jobs('state in %s', (NOT_DONE,)) as cr:
+                    for job_data in cr:
```
Comment (guewen) [inline queue_job/jobrunner/runner.py:391]:
```suggestion
                    for job_data in cr.fetchall():
```
?

Comment (sbidoul) [reply queue_job/jobrunner/runner.py:391]: @guewen why? My whole goal here is to fetch piecewise from the server.

**Review by ThomasBinsfeld — APPROVED** [review]
Functional test

File: queue_job/jobrunner/runner.py (hunk 2 above)
Comment (guewen) [reply queue_job/jobrunner/runner.py:391]: You are right :)

**Review by guewen — APPROVED** [review]

**Pull request merged**